config/coreboot-x200-maximized.config

```diff
@@ -0,0 +1,14 @@
+CONFIG_VENDOR_LENOVO=y
+CONFIG_CBFS_SIZE=0x7FCFFF
+CONFIG_IFD_BIN_PATH="@BLOB_DIR@/xxx0/ifd.bin"
+CONFIG_GBE_BIN_PATH="@BLOB_DIR@/xxx0/gbe.bin"
+CONFIG_HAVE_IFD_BIN=y
+CONFIG_BOARD_LENOVO_X200=y
+CONFIG_LINUX_COMMAND_LINE="intel_iommu=igfx_off quiet"
```

Comment:
This is Heads kernel optins passed from coreboot.
As of now, nothing is tweaked here for i915/drm driver and coreboot boots without IGFX_INIT, meaning that the kernel is seldom responsible for graphic initialization. 

linux config, as specified in board config, reuses linux-x230-legacy.conf, which includes i915 and i915 drm drivers. It should just work (tm).